cpp/data/system_updates/20.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
With the introduction of the new system_updates schema, scripts with ids <= 99 are no longer allowed. Please rename your script to `100.sh`